modules/clickhouse/src/main/java/org/testcontainers/containers/ClickHouseContainer.java

```diff
@@ -51,18 +51,18 @@ public ClickHouseContainer(final DockerImageName dockerImageName) {
         super(dockerImageName);
         dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
 
-        withExposedPorts(HTTP_PORT, NATIVE_PORT);
-        waitingFor(
-            new HttpWaitStrategy()
-                .forStatusCode(200)
-                .forResponsePredicate(responseBody -> "Ok.".equals(responseBody))
-                .withStartupTimeout(Duration.ofMinutes(1))
-        );
+        this.withExposedPorts(HTTP_PORT, NATIVE_PORT);
+        this.waitingFor(
```

Comment:
```suggestion
        withExposedPorts(HTTP_PORT, NATIVE_PORT);
        waitingFor(
```

Reply:
When I removed `this.` - IntelliJ Idea shows me a warning: `'SELF'` used without 'try'-with-resources statement`

Reply:
About support of both drivers - Clickhouse is now separate company, not part of Yandex. And when old driver is used:
```
2022-06-06T18:45:27.806+0900	WARN	main	ru.yandex.clickhouse.ClickHouseDriver	******************************************************************************************
2022-06-06T18:45:27.806+0900	WARN	main	ru.yandex.clickhouse.ClickHouseDriver	* This driver is DEPRECATED. Please use [com.clickhouse.jdbc.ClickHouseDriver] instead.  *
2022-06-06T18:45:27.807+0900	WARN	main	ru.yandex.clickhouse.ClickHouseDriver	* Also everything in package [ru.yandex.clickhouse] will be removed starting from 0.4.0. *
2022-06-06T18:45:27.807+0900	WARN	main	ru.yandex.clickhouse.ClickHouseDriver	******************************************************************************************
```
So I think its usage should be avoided.

Reply:
it complained about `this.` because the previous code for some reason did not pass a generic argument to `JdbcDatabaseContainer`, this should fix it:
https://github.com/testcontainers/testcontainers-java/pull/5474/files#r892189020